amundsen_application/jira/jira_client.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any
+from jira import JIRA, JIRAError, Issue
+from flask import current_app as app
+
+import logging
+
+SEARCH_STUB = 'project={project_id} AND text ~ "{table_key}"'
+
+
+class JiraClient:
+
+    def __init__(self) -> None:
+        self.jira_url = app.config['JIRA_URL']
+        self.jira_user = app.config['JIRA_USER']
+        self.jira_password = app.config['JIRA_PASSWORD']
+        self.jira_project_id = app.config['JIRA_PROJECT_ID']
```

Comment:
does it mean it could only search issue for single project instead of multiple project for same table?

Reply:
hm interesting point - i could remove this entirely and search across all projects. it would only return tickets the service account has access to so that seems probably fine. i removed it from the query string and will remove it as a config setting